src/formatter/Cli.hx

```diff
@@ -174,17 +177,20 @@ class Cli {
 				case Success(formattedCode):
 					Sys.println(formattedCode);
 					Sys.exit(0);
-				case Failure(_):
+				case Failure(errorMessage):
 					Sys.println(content);
+					Sys.stderr().writeString("Format failed: " + errorMessage);
 					Sys.exit(2);
 				case Disabled:
 					Sys.println(content);
+					Sys.stderr().writeString('Formatting is disabled in "${paths[0]}"');
```

Comment:
Hm... I feel like this one might not warrant an error message (or an exit code of 1), since this is the expected outcome if you have disabled formatting.

Sorry for all the nit-picking. ;)

Reply:
I've remove the message, but I'm keeping the exit code, so the caller can know that no formatting happened because it was disabled. Whether they use that information is up to them.